AppLovin MAX Demo App - Java/app/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
     compileSdkVersion 31
     defaultConfig {
         applicationId "com.applovin.enterprise.apps.demoapp"
-        minSdkVersion 16
+        minSdkVersion 19
```

Comment:
Wanted to leave a note about this. I had to adjust this in the code to use the google admob adapters. Not sure if we want to change this, so that developers can use the adapters but thought I would just point it out.

Reply:
I think we should keep it at 16 otherwise pubs might think our minimum SDK version for the AppLovin SDK is 19. If they want to add adapters they can always update this value themselves.